tests/trig/manifest.jsonld

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
No trig/manifest.ttl file.

Reply:
There's no turtle/manifest.ttl either, or sparql/manifest.ttl. The JSON-LD versions were there historically, and can be removed, as far as I'm concerned. I don't think the scripts rebuild them.